tools/git-version.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.ajoberstar:grgit:1.9.2'
+    }
+}
+
+import org.ajoberstar.grgit.Grgit
+
+ext {
+    git = Grgit.open(currentDir: projectDir)
+    gitVersionName = ""
+    gitVersionName = git.describe()
```

Comment:
In fact:

`git describe` will return a weird `v1.0.1-3-gf9b0032`

`git describe --tags` returns `v1.1.0`

So i think i will keep the `--tags` option